packages/web/src/containers/gainer-and-loser-container/GainerAndLoserContainer.tsx

```diff
@@ -108,7 +108,7 @@ const GainerAndLoserContainer: React.FC = () => {
   const { isLoadingCommon } = useLoading();
 
   const gainersList = useMemo(() => {
-    return gainers.map((item: IGainer) => {
+    return (gainers ?? [])?.map((item: IGainer) => {
```

Comment:
You can reduce the risk of out of index by reversing the order of the filter and map. 
Also, the check for `item.tokenPath === wugnotPath` could be better managed as a single variable.